src/Bit.BlazorUIPlayground.Web/Shared/MainLayout.razor

```diff
@@ -9,3 +9,18 @@
 
     <Footer />
 </main>
+
+@code {
+
+    [Inject] private IJSRuntime _js { get; set; }
+
+    protected override void OnAfterRender(bool firstRender)
+    {
+        if (firstRender)
+        {
+            BitThemeManager.init(_js);
+            var myTheme = new BitTheme();
+            BitThemeManager.ApplyBitTheme(myTheme);
```

Comment:
remove these 2 redundant lines of code. only the init call is required.

Reply:
Update to the latest pre release and use `Init` instead of `init`